packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx

```diff
@@ -90,8 +91,10 @@ export default function RecurringEventController({
               title={t("recurring_event")}
               {...recurringLocked}
               description={t("recurring_event_description")}
-              checked={recurringEventState !== null}
+              checked={!!recurringEventState}
               data-testid="recurring-event-check"
+              disabled={!recurringEventState && isSeatsOffered}
+              tooltip={isSeatsOffered ? t("seats_doesnt_support_recurring") : undefined}
```

Comment:
`recurring_event_seats_error` is a better message here because it tells the user what to do